### PR TITLE
Implement tests for scenario parser.

### PR DESCRIPTION
**Pull request by MiSobecki**

### CODE REVIEW

File: src/main/java/com/nokia/uwr/service/parser/ScenarioFileParserImpl.java [hunk 1]
```diff
@@ -23,15 +23,15 @@ public class ScenarioFileParserImpl implements ScenarioFileParser {
     @Override
     public ScenarioSchema parseJSONFile(File resource) {
         try {
-            LOGGER.info("Read value from JSON file");
+            LOGGER.info("Read value from JSON file: " + resource.getName());
 
             ScenarioSchema scenarioSchema = mapper.readValue(resource, ScenarioSchema.class);
 
-            LOGGER.info("Read value from JSON file successfully");
+            LOGGER.info("Read value from JSON file successfully: " + resource.getName());
```
Comment (PiotrStoklosa) [inline src/main/java/com/nokia/uwr/service/parser/ScenarioFileParserImpl.java]: I think that printing resource name one time (in 26 line) is enough.

File: src/test/java/com/nokia/uwr/service/parser/ScenarioFileParserImplTest.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+                        new UEStep(8, new Localization(24, 16), "end")))));
+
+        ScenarioSchema expected = new ScenarioSchema(bts, ues);
+
+        // when
+        ScenarioSchema returned;
+
+        try {
+            File resource = ResourceUtils.getFile(Objects.requireNonNull(
+                    this.getClass().getResource("/correctFile.json")));
```
Comment (PiotrStoklosa) [inline src/test/java/com/nokia/uwr/service/parser/ScenarioFileParserImplTest.java]: I think, extracting file name to static final variable is better idea

File: src/test/java/com/nokia/uwr/service/parser/ScenarioFileParserImplTest.java [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+        ues.add(new UEScenario("UE1", new ArrayList<>(
+                List.of(new UEStep(10, new Localization(10, 11), "start"),
+                        new UEStep(12, new Localization(10, 12), "move"),
+                        new UEStep(15, new Localization(10, 12), "end")))));
+        ues.add(new UEScenario("UE2", new ArrayList<>(
+                List.of(new UEStep(5, new Localization(20, 18), "start"),
+                        new UEStep(6, new Localization(22, 18), "move"),
+                        new UEStep(7, new Localization(24, 18), "move"),
+                        new UEStep(8, new Localization(24, 16), "end")))));
+
```
Comment (PiotrStoklosa) [inline src/test/java/com/nokia/uwr/service/parser/ScenarioFileParserImplTest.java:56]: List.of() is in my opinion better alternative to create list with given elements

File: src/test/java/com/nokia/uwr/service/parser/ScenarioFileParserImplTest.java [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+    }
+
+    @Test
+    public void shouldReturnNullWhileGivenIncorrectFileFormat() {
+        // when
+        ScenarioSchema returned;
+
+        try {
+            File resource = ResourceUtils.getFile(Objects.requireNonNull(
+                    this.getClass().getResource("/incorrectFile.json")));
```
Comment (PiotrStoklosa) [inline src/test/java/com/nokia/uwr/service/parser/ScenarioFileParserImplTest.java]: Same as above

File: src/test/java/com/nokia/uwr/service/parser/ScenarioFileParserImplTest.java [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+    @Test
+    public void shouldReturnCorrectScenarioSchemaInstanceWhileGivenCorrectFileFormat() {
+        // given
+        List<BTSDescription> bts = new ArrayList<>();
+        bts.add(new BTSDescription("BTS1", new Localization(1, 5), 40));
+        bts.add(new BTSDescription("BTS2", new Localization(20, 30), 38));
+
+        List<UEScenario> ues = new ArrayList<>();
+        ues.add(new UEScenario("UE1", new ArrayList<>(
+                List.of(new UEStep(10, new Localization(10, 11), "start"),
```
Comment (PiotrStoklosa) [inline src/test/java/com/nokia/uwr/service/parser/ScenarioFileParserImplTest.java]: For start/move/end we can create enum

File: src/main/java/com/nokia/uwr/service/parser/ScenarioFileParserImpl.java [hunk 6]
```diff
@@ -23,15 +23,15 @@ public class ScenarioFileParserImpl implements ScenarioFileParser {
     @Override
     public ScenarioSchema parseJSONFile(File resource) {
         try {
-            LOGGER.info("Read value from JSON file");
+            LOGGER.info("Read value from JSON file: " + resource.getName());
```
Comment (PiotrStoklosa) [inline src/main/java/com/nokia/uwr/service/parser/ScenarioFileParserImpl.java:29]: If resource == null, there will be NullPointerException

**Review by PiotrStoklosa — CHANGES_REQUESTED** [review]
Request changes I mention in comments

File: src/test/java/com/nokia/uwr/service/parser/ScenarioFileParserImplTest.java (hunk 3 above)
Comment (MiSobecki) [reply src/test/java/com/nokia/uwr/service/parser/ScenarioFileParserImplTest.java:56]: It is already implemented as You mentioned

**Pull request merged**